docs/tutorials/airflow-trigger-airbyte-job.md

```diff
@@ -0,0 +1,117 @@
+---
+description: Create a DAG in Airflow to trigger Airbyte sync job
+---
+
+# Airflow Integration 
+Airbyte allows to trigger the syncronization job using Operators in Airflow. This tutorials explain how to configure your DAG in Airflow to do this. 
+
+## 1. Set up the tools
+
+First of all, make sure you have Docker and Docker Compose installed. 
+
+### **Start Airbyte**
+For this tutorial you need have a connection created. If you are starting with Airbyte and want to test the integration, we suggest to you follow the [Getting Started tutorial](../getting-started.md) and use that connection.
+
+The recommendation is to set up the **sync frequency** of your Connection to **manual** because Airflow will become responsible to trigger this process.
```

Comment:
@marcosmarxm what happens if we don't set sync to manual? Will this cause problems?

Reply:
Airbyte will trigger automatic that sync usually. It's not a problem but can generate some confusion.

Think that you set the **sync frequency** to 1h in Airybte, but your Airflow job is `@daily`
And you have a pipeline e.g `airbyte_sync_job >> aggregate_data >> send_report_email` in Airflow.
When you start your daily job in Airflow it'd already triggered 23x in Airbyte.

Could be a problem with you have a dashboard where you use the raw data collected in the `airbyte_sync_job` and also the result from `aggregate_data`. Why?
In the dashboard, a graph using `airbyte_sync_job` has 1h delay, but another graph using the `aggregate_data` has D-1 delay because is only updated by Airflow.